### PR TITLE
render path for odp

### PR DESCRIPTION
**Pull request by sorja**
part of #1105 

ss: Left development, right production

image

todo
- [x] fix deepscan
- [x] fix url(s)
- [x] render components
- [x] weird issue where year jumps next to references

### CODE REVIEW

File: client/basePaths.ts [hunk 1]
```diff
@@ -18,29 +19,37 @@ export const BasePaths = {
...
     resetPassword: () => `/login/resetPassword`,
     invitation: () => '/login/invitation',
   },
+  OriginalDataPoint: {
+    root: (
+      countryIso: CountryIso | defaults.countryIso = defaults.countryIso,
+      assessmentName: AssessmentName | defaults.assessmentName = defaults.assessmentName,
+      cycleName = defaults.cycleName,
+      odpId = ':odpId'
+    ) => _generate(countryIso, assessmentName, cycleName, 'originalDataPoint', odpId),
```
Comment (sorja) [inline client/basePaths.ts:51]: should we prefix with assessment like other assessment paths?
if not, should we prefix instead with originalDataPoint?

currently looks like:
http://localhost:9000/FIN/fra/2025/originalDataPoint/72

Comment (minotogna) [reply client/basePaths.ts:51]: it should follow assessment convention. then yes.
and they should be: /:countryIso/assessments/:name/cycles/:cycleName/originalDataPoint/:odpId

Comment (minotogna) [reply client/basePaths.ts:51]: call the key, one, not root

Comment (sorja) [reply client/basePaths.ts]: I think we need to revisit the paths in general
For client side, I think 'root' makes more sense?
We also have other paths under OriginalDataPoint (section, tab)

Renamed for now

**Review by minotogna — CHANGES_REQUESTED** [review]
convert less to sass

**Review by minotogna — APPROVED** [review]

**Pull request merged**